### DISCUSSION TITLE
Deploy Map Leaflet R

### DISCUSSION THREAD

**Discussion by Samgoesniady** [teamalgoritma/community#443] [Q&A] [answered]
Hallo kak saya tadi mencoba mendeploy map yang telah saya coba buat di r markdown namun ketika saya coba buat menjadi dashboard, mapnya gagal solusinya bagaimana ya kak? Terima kasih
![IMG-20221106-WA0001](https://user-images.githubusercontent.com/100010379/200150301-b67614f5-408b-4b1c-b097-61b6e5ffe7e4.jpg)
![IMG-20221106-WA0002](https://user-images.githubusercontent.com/100010379/200150302-b2d16b0f-8e16-4814-94fd-1c7070f1a273.jpg)
![IMG-20221106-WA0000](https://user-images.githubusercontent.com/100010379/200150304-fe99531b-1bbe-466d-97fb-20d91636a1fa.jpg)

**Comment by kevbow** [comment, marked as answer]
Dear @Samgoesniady,

Pada baris pertama Anda menggunakan `renderPlotly` sementara object yang dibentuk adalah leaflet.
Silakan coba ganti kode tersebut menggunakan `renderLeaflet`.
Semoga membantu dan terima kasih.

**Reply by Samgoesniady** [reply]
Terima kasih kak sudah berhasil
